lib/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -858,6 +862,34 @@ namespace Opm {
     }
 
 
+    void Eclipse3DProperties::handleMAXVALUEKeyword( const DeckKeyword& deckKeyword, BoxManager& boxManager) {
```

Comment:
Please add a brief comment that the `MAXVALUE` and `MINVALUE` keywords are processed in place for the current value of the keyword, and do not "stick" as an attribute of the keyword.